### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by copyrighthero** [copyrighthero/MI-449-SS17-740-html-multiple-pages-T_njfg#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://zhaohans-kakudc9-j3bx8t_njfg.surge.sh/) and meets the following criteria:

- [x] None of the routes on the website end with `.html`
- [x] A homepage exists, where the purpose of the site is explained
- [x] A "list page" exists that lists your top 3 favorite of this thing (e.g. `/pokemon`) -- each list item should link to the corresponding page for that favorite, detailed below
- [x] For each of your favorites, a page exists to display more information about it (e.g. `/pokemon/bulbasaur`) -- we want to make sure you practice multiple levels of nested routes
- [x] Every page includes links to the homepage and list page (although pages do not have to link to themselves)

Have put a lot of time in it, I think I'll do all of my future projects based on it.

**Comment by stuartpearman** [comment]
Hey Hansheng,

the hosted link isn't working, make sure your project is hosted to surge. There should be a command in the project details to do that.

**Comment by copyrighthero** [comment]
@stuartpearman Hello, I have updated the github pages as well as the surge hosted site, thank you for your patience.

**Comment by stuartpearman** [comment]
Hi Hansheng, the site is looking pretty nice. It's not what we intended with the assignment, but it's not necessarily wrong. There are a few things I really want you to grasp:

### Naming routes

Read this criteria:

> For each of your favorites, a page exists to display more information about it (e.g. `/pokemon/bulbasaur`) -- we want to make sure you practice multiple levels of nested routes

I was able to find your nested routes, but I didn't really understand the naming conventions you were going for. A big part of the assignment is that each route describes what you'll find on the page. Look at the URL for this github issue:

    https://github.com/copyrighthero/MI-449-SS17-740-html-multiple-pages-T_njfg/issues/1

The first part is the root directory with the name of the site:

    https://github.com/

Then it has your username:

    copyrighthero/

Then the name of your repository:

    MI-449-SS17-740-html-multiple-pages-T_njfg/

Issues is a sub-directory within your repo:

    /issues

and this is the first issue:

    /1

When setting up your routes, think of what the names of each sub-directory mean in relation to their parent directories, and be descriptive if you can.

### Adding and subtracting content

It's totally fine that you used code from other assignments as a starting point for this assignment, people use boilerplates and templates all the time for making websites. But you also left a lot of the content and functionality in from previous assignments.

Think of it this way, if you were working off of a template and left in the placeholder content, your website would be a mess! The same goes for keeping content from past assignments. If you keep your code organized, you'll always be able to go back and get old code, and everything is on GitHub too 🙂  If you want to find ways to reuse your earlier work at the end of the class, that's totally fine, but in your assignments, I'd really like you to be selective about what you keep from project to project.

To summarize this section, reusing code from past projects is good! But it's as much about taking code out as it is about adding new code in. Make sure you are being just as deliberate when you reuse code as when you write it for the first time :+1:

**Comment by copyrighthero** [comment]
Hello @stuartpearman ,

With the latest commit, I have removed redundant codes, removed all third party js libs from the html files, and renamed the sub folders using descriptive words, could you please review?

Thank you and have a good day!
Hansheng

**Comment by stuartpearman** [comment]
Looks a lot better Hansheng! We'll revisit this concept later as we learn more server-side development. Let me know if you have any questions :shipit:
